release_util/management/commands/show_unapplied_migrations.py

```diff
@@ -54,7 +54,11 @@ def handle(self, *args, **kwargs):
 
         # Compose the output YAML.
         yaml_output = yaml.safe_dump(
-            {'migrations': unapplied, 'initial_states': current, 'database': kwargs['database']}
+            {
+                'migrations': [{'app': item[0], 'migration': item[1]} for item in unapplied],
+                'initial_states': [{'app': item[0], 'migration': item[1]} for item in current],
+                'database': kwargs['database']
+            }
```

Comment:
Also, `run_migrations_one_by_one.py` and the `MigrationSessionDeprecated` class count on this output - and either need to be changed *or* actually deleted.